sdk/billingbenefits/arm-billingbenefits/samples-dev/savingsPlanOrderGetSample.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { BillingBenefitsRP } from "@azure/arm-billingbenefits";
+import { DefaultAzureCredential } from "@azure/identity";
+
+/**
+ * This sample demonstrates how to Get a savings plan order.
+ *
+ * @summary Get a savings plan order.
+ * x-ms-original-file: specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/examples/SavingsPlanOrderGet.json
+ */
+async function savingsPlanOrderGet() {
+  const savingsPlanOrderId = "20000000-0000-0000-0000-000000000000";
+  const credential = new DefaultAzureCredential();
+  const client = new BillingBenefitsRP(credential);
+  const result = await client.savingsPlanOrder.get(savingsPlanOrderId);
+  console.log(result);
+}
+
+savingsPlanOrderGet().catch(console.error);
+
+/**
+ * This sample demonstrates how to Get a savings plan order.
+ *
+ * @summary Get a savings plan order.
+ * x-ms-original-file: specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/examples/SavingsPlanOrderExpandedGet.json
+ */
+async function savingsPlanOrderWithExpandedPaymentsGet() {
+  const expand = "schedule";
+  const savingsPlanOrderId = "20000000-0000-0000-0000-000000000000";
+  const credential = new DefaultAzureCredential();
+  const client = new BillingBenefitsRP(credential, expand);
```

Comment:
expand should not be here, it's a parameter of get()
swagger link:https://github.com/Azure/azure-rest-api-specs/blob/1fefe3f5cee88319b17c08a2dbf95e1e983a9f8c/specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/billingbenefits.json#L197